playground/named/c/webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const crypto = require('crypto');
+const { NamedChunksPlugin } = require('webpack');
+
+module.exports = {
+  entry: './src/index.js',
+  plugins: [
+    new NamedChunksPlugin((chunk) => chunk.name || crypto
+      .createHash('sha256')
+      .update(`${chunk.id}_${Math.random()}`)
```

Comment:
@Whoaa512 The question unrelated to the PR itself: do you use such a pattern in your codebase? I mean exactly this one, naming a chunk with a random value? If the answer is "yes", what for do you need that?

Reply:
😅 No I just wanted the chunks to have different names. I tried just using `chunk.id` but the ids are numbers so it didn't have enough entropy to be unique

Reply:
Alright, makes sense! :D